### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Ustimov**

### CODE REVIEW

File: jp.py [hunk 1]
```diff
@@ -239,6 +239,47 @@ def install(self):
 
         return changed
```
Comment (Ustimov) [inline jp.py]: E303 too many blank lines (5)

Comment (Ustimov) [inline jp.py]: E111 indentation is not a multiple of four

Comment (Ustimov) [inline jp.py]: E501 line too long (155 > 79 characters)

Comment (Ustimov) [inline jp.py]: E116 unexpected indentation (comment)

Comment (Ustimov) [inline jp.py]: E266 too many leading '#' for block comment

Comment (Ustimov) [inline jp.py]: E303 too many blank lines (6)

Comment (Ustimov) [inline jp.py]: E111 indentation is not a multiple of four

Comment (Ustimov) [inline jp.py]: E111 indentation is not a multiple of four

Comment (Ustimov) [inline jp.py]: E112 expected an indented block

Comment (Ustimov) [inline jp.py]: E113 unexpected indentation

Comment (Ustimov) [inline jp.py]: E266 too many leading '#' for block comment

Comment (Ustimov) [inline jp.py]: E303 too many blank lines (5)

Comment (Ustimov) [inline jp.py]: E303 too many blank lines (5)

File: jp.py [hunk 2]
```diff
@@ -239,6 +239,47 @@ def install(self):
...
 
+
+
```
Comment (Ustimov) [inline jp.py]: E303 too many blank lines (5)

File: jp.py [hunk 3]
```diff
@@ -239,6 +239,47 @@ def install(self):
...
+
+
```
Comment (Ustimov) [inline jp.py]: E111 indentation is not a multiple of four

**Comment by UPlatformTeam** [comment]
Metrics counted

**Comment by UPlatformTeam** [comment]
Metrics counted

**Pull request closed without merging**